docs/src/main/paradox/api-reference/schemas.md

```diff
@@ -1,252 +1,153 @@
-
-
-### Example request
-```bash
-curl -v -X PATCH "https://bbp-nexus.epfl.ch/{environment}/{version}/schemas/nexus/core/subject/v1.0.0/config?rev=2"
+
+### Publish a schema
+
+```
+PATCH /v0/schemas/{orgId}/{domId}/{name}/{version}/config?rev={rev}
 ```
 
+The json payload contains the key `published` with it's value (true|false).
 
-## Deprecate
-###**`DELETE /schemas/{orgId}/{domainId}/{name}/{version}?rev={rev}`**
-**Deprecates** the schema specified in **name & version** from the domain **domainId** inside the organization **orgId**.
+#### Example
 
-### Request path and query parameters
-| Field         | Type          | Description                                                                                                                   |
-| ------------- |-------------  | ---------------------------------------------                                                                                 |
-| orgId         | String        | The unique identifier for the organization.                                                                                   | 
-| domainId      | String        | The unique identifier for the domain for this particular organization.                                                        |
-| name          | String        | The unique identifier for the schema for this particular domain. It has to match the following regex: ([a-zA-Z0-9]+)          |
-| version       | String        | The unique identifier for the schema version for this particular schema, following the [semantic format](http://semver.org/). |
-| rev           | Long          | The current revision of the schema.                                                                                           |
+Request
+:   @@snip [schema-patch.sh](../assets/api-reference/schemas/schema-patch.sh)
 
-### Response JSON-LD Object
-| Field         | Type          | Description                                   |
-| ------------- |-------------  | --------------------------------------------- |
-| @id           | String        | The unique identifier for the schema.         |
-| rev           | Long          | The current revision of the schema.           |
+Payload
+:   @@snip [schema-patch.json](../assets/api-reference/schemas/schema-patch.json)
 
-### Status Codes
-* **200 OK** - Request completed successfully.
-* **400 Bad Request** - The schema, domain or organization are deprecated.
-* **409 Conflict** - The provided rev query parameter does not match the current revision.
-* **404 Not Found** - The schema, domain or organization do not exist.
+Response
+:   @@snip [schema-ref-new.json](../assets/api-reference/schemas/schema-ref-patch.json)
 
 
-### Example request
-```bash
-curl -v -X DELETE "https://bbp-nexus.epfl.ch/{environment}/{version}/schemas/nexus/core/subject/v1.0.0?rev=2"
+### Deprecate a schema
+
+```
+DELETE /v0/schemas/{orgId}/{domId}/{name}/{version}?rev={rev}
 ```
 
```

Comment:
Add:

Deprecating a schema prevents the creation of new instances that conform to it.